modules/persistent-storage-csi-cloning-using.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-cloning.adoc
+
+[id="persistent-storage-csi-cloning-using_{context}"]
+= Using a cloned PVC as a storage volume
+
+A newly cloned PVC can be consumed, cloned, snapshotted, or deleted independently of its original dataSource PVC.
+
+Pods can access storage by using the cloned PVC as a volume. For example:
+
+.Use CSI volume clone in the Pod
```

Comment:
I wanted to make this module (`Using`) a procedure, so I created the example definition. It's a modified and shorter version of https://docs.openshift.com/container-platform/4.3/storage/understanding-persistent-storage.html#pvc-claims-as-volumes_understanding-persistent-storage but not sure it fits here, or if it's accurate.